modules/br/module.json

```diff
@@ -20,7 +20,7 @@
 			"twitter": "@ForgeVTT"
 		}
 	],
-	"url": "https://www.dndbeyond.com/sources/BR",
+	"url": "https://www.dndbeyond.com/sources/dnd/br",
```

Comment:
This URL gets a 404. The correct URL is `https://www.dndbeyond.com/sources/dnd/basic-rules-2014`.